MetaMask/metamask-mobile
# fix: Change network id for chain id

## **Description**
We were using network id instead of chain id when resolving ens domains on the browser.
In this PR we start using the current chain Id instead of fetching the network id. All the behaviour remains the same.
## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this in app browser
2. Browse 'vitalik.eth'
3. should resolve as expected 

## **Screenshots/Recordings**
https://recordit.co/wuzC6p3Rnd

## E2E
https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/b5c88ca0-20f2-460e-98ec-da7173cbf687

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
